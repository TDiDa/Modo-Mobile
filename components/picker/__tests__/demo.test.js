import demoTest from '../../../tests/shared/demoTest';

demoTest('picker');
