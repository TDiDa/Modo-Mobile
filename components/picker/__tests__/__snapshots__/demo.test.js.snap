// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/picker/demo/basic.md correctly 1`] = `
<div
  class="m-picker"
>
  <div
    class="m-picker-column"
  >
    <div
      class="m-picker-column-mask"
    />
    <div
      class="m-picker-column-indicator"
    />
    <div
      class="m-picker-column-list"
    >
      <div
        class="m-picker-column-item"
      >
        2015
      </div>
      <div
        class="m-picker-column-item"
      >
        2016
      </div>
      <div
        class="m-picker-column-item"
      >
        2017
      </div>
      <div
        class="m-picker-column-item"
      >
        2018
      </div>
      <div
        class="m-picker-column-item"
      >
        2019
      </div>
    </div>
  </div>
  <div
    class="m-picker-column"
  >
    <div
      class="m-picker-column-mask"
    />
    <div
      class="m-picker-column-indicator"
    />
    <div
      class="m-picker-column-list"
    >
      <div
        class="m-picker-column-item m-picker-column-item-selected"
      >
        春
      </div>
      <div
        class="m-picker-column-item"
      >
        夏
      </div>
      <div
        class="m-picker-column-item"
      >
        秋
      </div>
      <div
        class="m-picker-column-item"
      >
        冬
      </div>
    </div>
  </div>
</div>
`;

exports[`renders ./components/picker/demo/cascade.md correctly 1`] = `
<div
  class="m-picker"
>
  <div
    class="m-picker-column"
  >
    <div
      class="m-picker-column-mask"
    />
    <div
      class="m-picker-column-indicator"
    />
    <div
      class="m-picker-column-list"
    >
      <div
        class="m-picker-column-item"
      >
        安徽省
      </div>
      <div
        class="m-picker-column-item m-picker-column-item-selected"
      >
        澳门特别行政区
      </div>
      <div
        class="m-picker-column-item"
      >
        北京
      </div>
    </div>
  </div>
  <div
    class="m-picker-column"
  >
    <div
      class="m-picker-column-mask"
    />
    <div
      class="m-picker-column-indicator"
    />
    <div
      class="m-picker-column-list"
    >
      <div
        class="m-picker-column-item"
      >
        澳门半岛
      </div>
      <div
        class="m-picker-column-item m-picker-column-item-selected"
      >
        离岛
      </div>
    </div>
  </div>
  <div
    class="m-picker-column"
  >
    <div
      class="m-picker-column-mask"
    />
    <div
      class="m-picker-column-indicator"
    />
    <div
      class="m-picker-column-list"
    />
  </div>
</div>
`;

exports[`renders ./components/picker/demo/popup.md correctly 1`] = `
<div>
  <div
    style="padding:10px 16px"
  >
    <span>
      自定义弹出
    </span>
    <span>
      : 
    </span>
    <span>
      2015 春
    </span>
  </div>
  <div
    class="m-popup m-popup-bottom"
  />
  <div
    class="m-list"
  >
    <div
      class="m-list-body"
    >
      <div
        class="m-list-item m-list-item-right"
      >
        <div
          class="m-list-item-inner"
        >
          <div
            class="m-list-item-label"
          >
            <div
              class="m-list-item-title"
            >
              选择时间
            </div>
          </div>
          <div
            class="m-list-item-content"
          />
        </div>
      </div>
      <div
        class="m-popup m-popup-bottom"
      />
      <div
        class="m-list-item m-list-item-disabled m-list-item-right"
      >
        <div
          class="m-list-item-inner"
        >
          <div
            class="m-list-item-label"
          >
            <div
              class="m-list-item-title"
            >
              禁用选择
            </div>
          </div>
          <div
            class="m-list-item-content"
          >
            2015 春
          </div>
        </div>
      </div>
      <div
        class="m-popup m-popup-bottom"
      />
      <div
        class="m-list-item m-list-item-right"
      >
        <div
          class="m-list-item-inner"
        >
          <div
            class="m-list-item-label"
          >
            <div
              class="m-list-item-title"
            >
              选择城市
            </div>
          </div>
          <div
            class="m-list-item-content"
          >
            澳门特别行政区 离岛
          </div>
        </div>
      </div>
      <div
        class="m-popup m-popup-bottom"
      />
    </div>
  </div>
</div>
`;
